Makefile

```diff
@@ -8,10 +8,10 @@ lint:
 
 test:
 	make lint
-	make cover
+	npm run test
 	
 cover:
-	node_modules/.bin/istanbul cover  node_modules/.bin/_mocha -- test --recursive --timeout=3000
+	make test
```

Comment:
Sorry but something seems a bit off here, especially the mix with `make` and `npm` commands, here is a good example taken from one of our repo :

```
lint:
	npm run lint

test:
	make lint
	make cover

cover:
	node_modules/.bin/istanbul cover node_modules/.bin/_mocha -- test --recursive
```